agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/job-configuration.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.exoplaform.org/xml/ns/kernel_1_2.xsd http://www.exoplaform.org/xml/ns/kernel_1_2.xsd"
+   xmlns="http://www.exoplaform.org/xml/ns/kernel_1_2.xsd">
+
+   <external-component-plugins>
+     <target-component>org.exoplatform.services.scheduler.JobSchedulerService</target-component>
+     <component-plugin>
+       <name>AddCronJob</name>
+       <set-method>addCronJob</set-method>
+       <type>org.exoplatform.services.scheduler.CronJob</type>
+       <description>add a cron job to the JobSchedulerService</description>
+       <init-params>
+         <properties-param>
+           <name>cronjob.info</name>
+           <description>details of the execution of the web reminder job</description>
+           <property name="jobName" value="ComputingReminderJob"/>
+           <property name="groupName" value="CollaborationSuite"/>
+           <property name="job" value="org.exoplatform.agenda.job.ComputingReminderJob"/>
+           <property name="expression" value="0 24/1 11 ? * * *"/>
```

Comment:
please comment on the frequency configured in this cron expression. Also, shouldn't we expose it as a property in exo.properties ? (maybe not, just asking).